cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -165,15 +172,15 @@ runTransactionBuildCmd
 
   txOuts <- mapM (toTxOutInAnyEra eon) txouts
 
-  -- Conway related
   votingProcedures <-
-    inEonForEra
-      (pure emptyVotingProcedures)
-      (\w -> firstExceptT TxCmdVoteError $ ExceptT (readVotingProceduresFiles w voteFiles))
-      eon
+    liftMaybeFeatured mfVoteFiles $ \w voteFiles ->
+      lift (readVotingProceduresFiles w voteFiles)
+        & onLeft (left . TxCmdVoteError)
```

Comment:
Using the new `liftMaybeFeatured` function to make it easy to manipulate the value within the `Featured` type.

`readVotingProceduresFiles` takes a `[VoteFile In]` and returns a `VotingProcedures era`

`liftMaybeFeatured` takes that computation to convert a `Maybe (Featured ConwayEraOnwards [VoteFile In])` to a `Maybe (Featured ConwayEraOnwards [VotingProcedures era])`